to-do-list/src/components/Interfaces/interfaces.tsx

```diff
@@ -0,0 +1,47 @@
+export interface PostItemProps{
```

Comment:
Don't save all the interfaces in the one file. Also, interfaces should be places in the src/ folder, not in the components/. Ind it should start from the lowercase to be consistent. As well as API - don't use uppercase in the src/ folders names, it's better to rename it to services/